source/beamline/BioXAS/BioXASSideTableCoordinator.cpp

```diff
@@ -157,24 +147,20 @@ void BioXASSideTableCoordinator::onAllControlsConnected(bool connected){
 		qDebug() << "Checking start up value from the OLD Horizontal DownStream status as " << horizontalDownstreamStatusControl_->value();
 		onHorizontalStatusControlValueChanged();
 
+		softIOCHeightControl_->move(softIOCHeightFeedbackControl_->value());
```

Comment:
Do these lines cause a motor to move or is this merely updating the PVs so that everything is sync'ed up?